soda/core/tests/data_source/test_group_evolution.py

```diff
@@ -27,3 +27,29 @@ def test_group_evolution(data_source_fixture: DataSourceFixture):
     scan.execute()
 
     scan.assert_all_checks_pass()
+
+
+@pytest.mark.skipif(
+    test_data_source not in ["postgres", "bigquery", "spark_df", "oracle"],
```

Comment:
You can do this using the datasource casify methods like here https://github.com/sodadata/soda-core/blob/c510e0b1a65ad13c0852d1a4decd86483a31b229/soda/core/tests/data_source/test_for_each_dataset.py#L43-L57

Reply:
Fixed.